lib/naver/searchad/api/ad-keyword/service.rb

```diff
@@ -7,7 +7,7 @@ module AdKeyword
         class Service < Naver::Searchad::Api::Core::BaseService
 
           def initialize
-            super('https://api.naver.com/', 'ncc/')
+            super('https://api.searchad.naver.com/', 'ncc/')
```

Comment:
maybe this url could be in some kind of a constant?

Reply:
I agree and it could, but refactoring is not part of this ticket. So maybe another time as it will require more testing otherwise.